src/app/dashboard/Missions/DetailsView.jsx

```diff
@@ -57,6 +57,12 @@ const useStyles = makeStyles((theme) => ({
   foodBoxDetailName: {
     padding: theme.spacing(1),
   },
+  buttonBox: {
+    spacing: theme.spacing(1),
+    display: "flex",
+    justifyContent: "center",
+    cursor: "pointer", // TODO: We shouldn't have to do this, figure out why
```

Comment:
If you don't want to force a cursor change, the element you use needs to be an `anchor` or `button` element. something naturally clickable. But in this case you're taking an mui `Box` for this

Reply:
Ok, switched to Buttons